packages/dataviews/src/validation.ts

```diff
@@ -34,6 +42,8 @@ export function isItemValid< Item >(
 				( field.type === 'email' && isEmptyNullOrUndefined( value ) ) ||
 				( field.type === 'integer' &&
 					isEmptyNullOrUndefined( value ) ) ||
+				( field.type === 'array' &&
+					isNotArrayEmptyOrContainsInvalidElements( value ) ) ||
```

Comment:
This is a small nit. For a second, I was confused for the use of "invalid elements" here, I thought _why are we validating isValid.elements here if there's a specific check for it_? Perhaps something along these lines makes that more clear?

```suggestion
					isArrayOrElementsEmptyNullOrUndefined( value ) ) ||
```